Microsoft.Research/ClousotMain/ClousotMain.cs

```diff
@@ -795,6 +796,10 @@ private bool FilterPreconditionsAccordingToWarningLevel(BoxedExpression exp)
 
       public int Analyze()
       {
+        // String formatting in the analyzer does not specify a culture, therefore some tests fail on systems with non-English culture with e.g. "expected: 2.0, actual: 2,0"
+        // Setting the threads culture should be removed as soon as issue #149 is fixed.
+        Thread.CurrentThread.CurrentCulture = CultureInfo.InvariantCulture;
```

Comment:
:bulb: What about using invariant culture if `/regression` option is used? Possible implementation: 4c98ec5a4e181a9d05b627f04c56507317d0c3ec
I think a more complicated `/LCID` does not add much value (why would you use it other than for this?).